docs/GETTING_STARTED.rst

```diff
@@ -0,0 +1,274 @@
+
+-  Install `s3cmd <http://s3tools.org/download>`__
+-  Install `redis <https://redis.io/download>`__ and start Redis.
+-  Add localCache section to your ``config.json``:
+
+::
+
+    "localCache": {
+        "host": REDIS_HOST,
+        "port": REDIS_PORT
+    }
+
+where ``REDIS_HOST`` is your Redis instance IP address (``"127.0.0.1"``
+if your Redis is running locally) and ``REDIS_PORT`` is your Redis
+instance port (``6379`` by default)
+
+-  Add the following to the etc/hosts file on your machine:
+
+.. code:: shell
+
+    127.0.0.1 bucketwebsitetester.s3-website-us-east-1.amazonaws.com
+
+-  Start the S3 server in memory and run the functional tests:
+
+.. code:: shell
+
+    npm run mem_backend
+    npm run ft_test
+
+Configuration
+-------------
+
+There are three configuration files for your Scality S3 Server:
+
+1. ``conf/authdata.json``, described above for authentication
+
+2. ``locationConfig.json``, to set up configuration options for
+
+   where data will be saved
+
+3. ``config.json``, for general configuration options
+
+Location Configuration
+~~~~~~~~~~~~~~~~~~~~~~
+
+You must specify at least one locationConstraint in your
+locationConfig.json (or leave as pre-configured).
+
+You must also specify 'us-east-1' as a locationConstraint so if you only
+define one locationConstraint, that would be it. If you put a bucket to
+an unknown endpoint and do not specify a locationConstraint in the put
+bucket call, us-east-1 will be used.
+
+For instance, the following locationConstraint will save data sent to
+``myLocationConstraint`` to the file backend:
+
+.. code:: json
+
+    "myLocationConstraint": {
+        "type": "file",
+        "legacyAwsBehavior": false,
+        "details": {}
+    },
+
+Each locationConstraint must include the ``type``,
+``legacyAwsBehavior``, and ``details`` keys. ``type`` indicates which
+backend will be used for that region. Currently, mem, file, and scality
+are the supported backends. ``legacyAwsBehavior`` indicates whether the
+region will have the same behavior as the AWS S3 'us-east-1' region. If
+the locationConstraint type is scality, ``details`` should contain
+connector information for sproxyd. If the locationConstraint type is mem
+or file, ``details`` should be empty.
+
+Once you have your locationConstraints in your locationConfig.json, you
+can specify a default locationConstraint for each of your endpoints.
+
+For instance, the following sets the ``localhost`` endpoint to the
+``myLocationConstraint`` data backend defined above:
+
+.. code:: json
+
+    "restEndpoints": {
+         "localhost": "myLocationConstraint"
+    },
+
+If you would like to use an endpoint other than localhost for your
+Scality S3 Server, that endpoint MUST be listed in your
+``restEndpoints``. Otherwise if your server is running with a:
+
+-  **file backend**: your default location constraint will be ``file``
+
+-  **memory backend**: your default location constraint will be ``mem``
+
+Endpoints
```

Comment:
I would put the "Endpoints" section" as a sub-title to the "Configuration" section, just like "Location Configuration"